CLAUDE.md

```
@@ -0,0 +1,95 @@
+## Purpose
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)